CompilerSource/ipc/GameInformation.fbs

```diff
@@ -0,0 +1,18 @@
+
+table GameInformation {
+  background_color:int;
+  embed_game_window:bool;
+  form_caption:string;
+  left:int;
+  top:int;
+  width:int;
+  height:int;

```

Comment:
Notice this uses left, top, width, and height, whereas the SpriteBoundingBox uses left, top, right, and bottom. Maybe we could reuse the same BoundingBox here and just have Sprite wrap around it to add its "Type"?